Dockerfile

```diff
@@ -0,0 +1,48 @@
+# RethinkRobotics/baxter-simulator:kinetic
```

Comment:
To learn docker I recommend Docker's basic tutorials, or you could skip that and go straight to the [ROS](http://wiki.ros.org/docker/Tutorials/Docker) ones or [moveit](http://moveit.ros.org/install/docker/) ones